File: solutions/csharp/restaurantmanagementsystem/Staff.cs
namespace RestaurantManagementSystem
{
    public class Staff
    {
        public int Id { get; }
        public string Name { get; }
        public string Role { get; }
        public string ContactNumber { get; }

        public Staff(int id, string name, string role, string contactNumber)
        {
            Id = id;
            Name = name;
            Role = role;
            ContactNumber = contactNumber;
        }
    }
}

File: solutions/java/src/restaurantmanagementsystem/model/Staff.java
package restaurantmanagementsystem.model;

public abstract class Staff {
    protected String id;
    protected String name;

    public Staff(String id, String name) {
        this.id = id;
        this.name = name;
    }

    public String getId() { return id; }
    public String getName() { return name; }
}